DemoApp/app/src/main/res/drawable/button_video.xml

```diff
@@ -2,8 +2,8 @@
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
     <item
         android:state_focused="true"
-        android:drawable="@drawable/button_video_focused" />
+        android:drawable="@drawable/button_random_focused" />
```

Comment:
Rename the drawable please. There should be no reference to random ad whatsoever.